tools/bgpreader.c

```diff
@@ -369,6 +391,16 @@ int main(int argc, char *argv[])
       fprintf(stderr, "INFO: Processing at most %d records\n", rec_limit);
       break;
 
+#ifdef WITH_RPKI
+    case 'H':      
+      rpki_input = bgpstream_rpki_parse_input(optarg);
+      if(!rpki_input->rpki_args_check){
+        bgpstream_rpki_destroy_input(rpki_input);
+        usage(); exit(-1);
```

Comment:
for the `exit(-1)` here, should it be `goto err;`, as in other parameters?

Reply:
In ```err:``` the RPKI config and input will be destroyed in case an error occurs while BGPStream is running through the data. 

While parsing the input the config does not exist. Therefore the RPKI input should be destroyed only.

Reply:
I see. That makes sense!